### ISSUE TITLE
Latest version breaks male height slider.

### ISSUE THREAD

**Issue by DirtyDarkling** [ManlyMarco/Illusion-Overlay-Mods#15]
Been trying to figure out what's been going on for a while now.  I finally figured out that the latest version of Overlay Mods seems to interfere with the Unlock Player Height mod, somehow.  Using 6.0.1.0 of Overlay mod makes the male height adjustment slider disappear in the character maker.  It seems only the slider is effected, all my old characters that had their heights changed before are still at their correct heights, I just can't make any height adjustments anymore.

![5 2 1 0](https://user-images.githubusercontent.com/64672234/118895763-c99cb500-b8d4-11eb-883a-5bf9a2441660.png)
![6 0 1 0](https://user-images.githubusercontent.com/64672234/118895769-cbff0f00-b8d4-11eb-8ac9-36d40f50964d.png)

**Comment by ManlyMarco** [comment]
It's a known issue, nothing in the update should cause this, it's agreed to be an issue in Unlock Player Height mod but no one cared to fix it so far.

**Comment by DirtyDarkling** [comment]
Ok, I didn't know it was already a known Issue.  I was pulling my hair out trying to figure out what had happened; I don't create male characters often so I just didn't notice back when I updated.  I thought it must of been one of the more recent mods I had installed that broke it.   

Weird that 5.2.1.0 works fine though.  To preface, I know nothing about coding, just curious. Is it that Unlock Player Height is using some older standard of UI injection that your update is overwriting or something?
